pkgs/development/libraries/gtkd/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchzip, atk, cairo, dmd, gdk_pixbuf, glib, gnome3, gst_all_1
+, librsvg, pango, pkgconfig, substituteAll, which }:
+
+stdenv.mkDerivation rec {
+  name = "gtkd-${version}";
+  version = "3.6.1";
+
+  src = fetchzip {
+    url = "https://gtkd.org/Downloads/sources/GtkD-${version}.zip";
+    sha256 = "1pnpq5jj95ba0yj77ly294hgbxwm44layxphcwbwpr2pxibfda9k";
+    stripRoot = false;
+  };
+
+  nativeBuildInputs = [ dmd pkgconfig which ];
+  propagatedBuildInputs = with gnome3; with gst_all_1; [
+    atk cairo gdk_pixbuf glib gstreamer gst-plugins-base gtk3 gtksourceview
+    libgda libpeas librsvg pango vte
+  ];
+
+  installFlags = "prefix=$(out)";
+
+  passthru = {
+    ldLibraries = propagatedBuildInputs;
```

Comment:
I initially passed `gtkd.propagatedBuildInputs` to `makeLibraryPath` in the tilix package, but it couldn't generate paths from the `out` output.